package.json

```diff
@@ -14,6 +14,7 @@
     "@babel/preset-typescript": "^7.1.0",
     "@commitlint/cli": "^8.1.0",
     "@commitlint/config-conventional": "^8.1.0",
+    "@cypress/webpack-preprocessor": "^4.1.0",
```

Comment:
This dependency allows for Cypress spec files to support imports in cypress files